### ISSUE TITLE
[Feature] bitcart-odoo/17.2

### ISSUE THREAD

**Issue by yonacwy** [bitcart/bitcart-odoo#1]
Hello,

Here is what bitcart-odoo17.2 community edition fork looks like, added some more menus and views for quicker access to links.

Still need to add some js scripts for the iframe  hcapcha to get the /admin login working from within odoo, and to cleanup the code. 

But the idea is to get bitcart inside of odoo working more like a app than a plugin without alot of work linking api's if possible, the menu is mostly placeholder ideas except for Settings.  

![Screenshot from 2023-12-24 12-19-09](https://github.com/bitcart/bitcart-odoo/assets/3211998/7445d5de-1124-43e7-a0b3-62b2b817455c)
![Screenshot from 2023-12-24 12-19-19](https://github.com/bitcart/bitcart-odoo/assets/3211998/bf7fcbdf-3275-4de6-92ea-325af2928eb1)
![Screenshot from 2023-12-24 12-19-29](https://github.com/bitcart/bitcart-odoo/assets/3211998/a9f17195-1f8a-4ec3-a07e-23d11c1b42fd)
![Screenshot from 2023-12-24 12-19-40](https://github.com/bitcart/bitcart-odoo/assets/3211998/4066de56-e518-4766-91f0-860d1f10ba1b)

**Comment by MrNaif2018** [comment]
Hi! I don't think embedding the website into iframe is that useful. Is it to provide more info? It has some security implications + I don't think we can't just use a few more description paragraphs. Is bitcart broken on odoo 17? Or you mean I should probably publish a new branch to support 17.x series?

**Comment by yonacwy** [comment]
I agree iframe is just quick testing without the js scripts for browser verification and keys for hcapcha but not much different than using a wrapper to other sources without signature verifications, the usefulness depends on the amount of time one spends on one platform or the other. 

I didn't see any bitcart odoo support for v17 the version im using so i forked and upgraded it myself to v17 to test it.
